frontstage/views/secure_messaging.py

```diff
@@ -74,23 +74,40 @@ def get_collection_case(party_id):
     return collection_case
 
 
+def get_survey_id(party_id):
```

Comment:
I'm actually thinking that we should have a little more logging on these functions. Maybe log that you're trying to retrieve this data then also log out the survey name at the end. Both probably debug